src/main/resources/templates/gardens/view.html

```diff
@@ -29,7 +31,8 @@ <h4>Owner </h4>
         </div>
     </div>
     <div class = "row  justify-content-center">
```

Comment:
if you do want a footer, when I was testing things I had to add style="margin-bottom:20px;" to this line to give some space for the footer

Reply:
Ok, I'll try that.  I must have deleted that by accident?  Not sure, but I'll add it in.